src/wp-includes/rest-api/endpoints/class-wp-rest-block-types-controller.php

```diff
@@ -705,6 +706,19 @@ public function get_item_schema() {
 				),
 				'keywords'              => $keywords_definition,
 				'example'               => $example_definition,
+				'block_hooks'           => array(
+					'description'       => __( 'This block is automatically inserted near any occurence of the block types used as keys of this map, into a relative position given by the corresponding value.' ),
+					'type'              => 'object',
+					'patternProperties' => array(
+						'^[a-zA-Z0-9-]+/[a-zA-Z0-9-]+$' => array(
+							'type' => 'string',
+							'enum' => array( 'before', 'after', 'first_child', 'last_child' ),
+						),
+					),
+					'default'           => array(),
+					'context'           => array( 'embed', 'view', 'edit' ),
+					'readonly'          => true,
+				),
```

Comment:
I've updated this based on https://github.com/WordPress/gutenberg/blob/8c5dbabe5361e94f18e4a55070db9a54d3c59d0b/lib/compat/wordpress-6.4/block-hooks.php#L341-L351. Hope that makes sense!

Reply:
I don't think that is correct and should changed in gutenberg. This pattern is not needed and make it extremely hard to read.

Reply:
See [my other comment](https://github.com/WordPress/wordpress-develop/pull/5203#discussion_r1325709612) 😄